manga compiling script.py

```diff
@@ -1,92 +0,0 @@
-import os
-import shutil
-import tempfile
-from pathlib import Path
-
-final_dir_name = str(input('What is the name and volume of the manga you '
-                         'are compiling (ex: chainsaw man volume 1): '))
-
-# final_dir_name = final_dir_name.replace(" ", "-")
-# final_dir_name = "Chainsaw Man Volume " + final_dir_name
-
-# get basic download path to access downloaded files
-downloads_path = str(Path.home() / "Downloads")
-
-# make new directory path in downloads from given name and volume
-final_dir_path = os.path.join(downloads_path, final_dir_name)
-
-# makes a new directory from new path - kinda unnecessary
-os.mkdir(final_dir_path)
-
-
-# self-explanatory; choose what chapters you want in said volume
-chapter_start = int(input('What Chapter are you starting at: '))
-chapter_end = int(input('What Chapter are you ending at: '))

```

Comment:
Do you ever need to make a cbz with a subset of chapters in the volume, as opposed to just all of them?